src/view/screens/Search/Search.tsx

```diff
@@ -876,6 +914,17 @@ function scrollToTopWeb() {
   }
 }
 
+// takes a bool. returns true once the bool is true, and stays true forever
+function useHasBeenTrue(bool: boolean) {
+  const [isTrue, setIsTrue] = React.useState(bool)
+
+  React.useEffect(() => {
+    if (bool) setIsTrue(true)
```

Comment:
In general pattern like this should be avoided. See here: https://react.dev/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes

We could fix this, but I wonder if it makes sense to just leave `enabled` on only for the current one? Same approach we do in `Home` and `Profile`. I'd like to at least be consistent with behavior with those — IMO if we change it here, we should do it as a single change across all surfaces (in a separate PR).

Reply:
Yeah, it's actually fine if the other tabs' queries are disabled when inactive. Removed